static/docs/commands-reference/root.md

```diff
@@ -12,7 +12,7 @@ usage: dvc root [-h] [-q | -v]
 
 While in project's sub-directory, sometimes developers may want to refer some
 file belonging to another directory. This command returns relative path to the
-DVC project's root directory from the current working directory. So, this
```

Comment:
`current working directory` or cwd is a know term

Reply:
Hmmm my life has been a lie! I always thought the standard term was "present working directory" from `pwd` but it seems the P there means "print"! Will revert all these changes back to "current"...

Reply:
Fixed in bc80b3d